### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by jlrgore**
yo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # foo
+making your shitty readme better. i type what i hear.
```
Comment (dlesaca) [inline README.md]: Thank god for you

**Comment by dlesaca** [comment]
Almost but not quite

File: README.md (hunk 1 above)
Comment (dlesaca) [inline README.md]: Fix your capitalization

Comment (jlrgore) [inline README.md]: my shift key is broken

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # foo
+MAKING YOUR SHITTY README BETTER.
```
Comment (dlesaca) [inline README.md]: Very good

**Pull request merged**